### PR TITLE
Add filter command

### PR DESCRIPTION
**Pull request by zacchaeuschok**
Allows users to filter by tag name. Might have to change how the filter function works if tag enums is implemented.

### CODE REVIEW

File: src/main/java/seedu/condonery/logic/parser/CondoneryParser.java [hunk 1]
```diff
@@ -6,15 +6,7 @@
...
-import seedu.condonery.logic.commands.AddCommand;
-import seedu.condonery.logic.commands.ClearCommand;
-import seedu.condonery.logic.commands.Command;
-import seedu.condonery.logic.commands.DeleteCommand;
-import seedu.condonery.logic.commands.EditCommand;
-import seedu.condonery.logic.commands.ExitCommand;
-import seedu.condonery.logic.commands.FindCommand;
-import seedu.condonery.logic.commands.HelpCommand;
-import seedu.condonery.logic.commands.ListCommand;
+import seedu.condonery.logic.commands.*;
```
Comment (lkwlkww) [inline src/main/java/seedu/condonery/logic/parser/CondoneryParser.java]: For this part, I recall that the coding standards taught by the module tell us to always import classes explicitly. What do you think?

**Review by lkwlkww — APPROVED** [review]
LGTM otherwise!

File: src/main/java/seedu/condonery/logic/parser/CondoneryParser.java (hunk 1 above)
Comment (zacchaeuschok) [reply src/main/java/seedu/condonery/logic/parser/CondoneryParser.java]: Oh yes, the auto-import made the change. I have fixed it. Thanks!

**Review by yuehernkang — APPROVED** [review]
LGTM

**Pull request merged**